gcc/rust/expand/rust-macro-builtins.cc

```diff
@@ -576,10 +576,8 @@ MacroBuiltin::include_str_handler (location_t invoc_locus,
   if (expect_single)
     rust_error_at (invoc_locus, "%s was not a valid utf-8 file",
 		   target_filename.c_str ());
-  else if (!bytes.empty ())
-    str = std::string ((const char *) &bytes[0], bytes.size ());
   else
-    return tl::nullopt;
```

Comment:
It's not strictly equivalent, as `nullopt` can be used to signal that the transcriber didn't produce anything. Maybe that doesn't really matter, or it's better to have an empty fragment created here with sloc info. @CohenArthur ?